Projects/UOContent/Spells/Eighth/Resurrection.cs

```diff
@@ -26,26 +27,30 @@ public ResurrectionSpell(Mobile caster, Item scroll = null) : base(caster, scrol
 
         public void Target(Mobile m)
         {
-            if (m == Caster)
+            if (!Caster.Alive)
+            {
+                Caster.SendLocalizedMessage(501040); // The resurrecter must be alive.
+            }
+            else if (m == Caster)
             {
                 Caster.SendLocalizedMessage(501039); // Thou can not resurrect thyself.
             }
```

Comment:
Since you can't cast spells while alive, you can't resurrect yourself anyways, so this should never trigger either? 🤔